BitList.cs

```diff
@@ -0,0 +1,1744 @@
+using System;
```

Comment:
Codacy found an issue: [This file has 1135 lines, which is greater than 1000 authorized. Split it into smaller files.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=5559704)